xml/containers-support.xml

```diff
@@ -0,0 +1,222 @@
+    the same packages as the main &slsa; &productnumber; release.
+   </para>
+  </listitem>
+ </orderedlist>
+ <sect1 xml:id="id-supported-containers-on-suse-host-environments">
+  <title>Supported Containers on SUSE Host Environments</title>
+
+  <para>
+   The following support options (Tiers) apply to
+   <emphasis role="strong">&sls;</emphasis> and
+   <emphasis role="strong">&caasp;</emphasis> containers on &suse; host
+   environments.
+  </para>
+
+  <sect2 xml:id="id-tier-one">
+   <title>Tier One</title>
+   <variablelist>
+    <varlistentry>
+     <term>Containers Delivered by &suse;</term>
+     <listitem>
+      <para>
+       Containers delivered by &suse; are fully supported. This applies to both
+       the container and host environment as well as all products under
+       support, applying to both general support as well as
+       <link xlink:href="https://www.suse.com/products/long-term-service-pack-support/">Long
+       Term Service Pack Support</link> (LTSS).
+      </para>
+     </listitem>
+    </varlistentry>
+   </variablelist>
+  </sect2>
+
+  <sect2 xml:id="id-tier-two">
+   <title>Tier Two</title>
+   <variablelist>
+    <varlistentry>
+     <term>Containers Delivered by Partners with an Agreement Ensuring a Joint Engineering Collaboration</term>
+     <listitem>
+      <para>
+       This Tier targets important Independent Software Vendors (ISVs). Partner
+       containers with a joint engineering collaboration agreement are fully
+       supported. This applies to both the container and host environment as
+       well as all products under support (both general, as well as LTSS)
+       covered by the agreement. Products not covered by the agreement fall
+       under Tier Three.
+      </para>
+     </listitem>
+    </varlistentry>
+   </variablelist>
+  </sect2>
+
+  <sect2 xml:id="id-tier-three">
+   <title>Tier Three</title>
+   <variablelist>
+    <varlistentry>
+     <term>Any Other Third Party Containers</term>
+     <listitem>
+      <para>
+       The &suse; container host environment is fully supported, however issues
+       with the third party containers themselves are to be brought up to the
+       given container vendor.
+      </para>
+     </listitem>
+    </varlistentry>
+   </variablelist>
+  </sect2>
+ </sect1>
+ <sect1 xml:id="id-supported-container-host-environments">
+  <title>Supported Container Host Environments</title>
+
+  <para>
+   The following support options (Tiers) apply to:
+  </para>
+
+  <itemizedlist>
+   <listitem>
+    <para>
+     &sle; base containers
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Containers delivered by &suse;
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Third party container environments derived from the &slea; base containers
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Third party containers built from &slea; packages using Kiwi
```

Comment:
* "Third-party" needs a hyphen.
* There should be an entity for KIWI. Otherwise, just capitalize it as "KIWI".